### PR TITLE
Filter empty LUGs

### PR DESCRIPTION
**Pull request by casperboone**
This PR:
* Filters LUGs with only returns
* Fixes assertions in the LUG integration test
* Changes `cfg` to `lug` in the integration test

### CODE REVIEW

File: modules/pipeline/jimple-library-usage-graph-generator/src/main/kotlin/org/cafejojo/schaapi/pipeline/usagegraphgenerator/jimple/ControlFlowGraphGenerator.kt [hunk 1]
```diff
@@ -16,25 +15,25 @@ object ControlFlowGraphGenerator {
      * Creates the control flow graph of a method body.
      *
      * @param body a Soot method [Body]
-     * @return the [Body]'s root [Unit] wrapped in a [Node]
+     * @return the [Body]'s root [Unit] wrapped in a [JimpleNode]
```
Comment (gandreadis) [inline modules/pipeline/jimple-library-usage-graph-generator/src/main/kotlin/org/cafejojo/schaapi/pipeline/usagegraphgenerator/jimple/ControlFlowGraphGenerator.kt]: Shouldn't `Unit` then also be a `Stmt`?

File: modules/pipeline/jimple-library-usage-graph-generator/src/test/kotlin/org/cafejojo/schaapi/pipeline/usagegraphgenerator/jimple/IntegrationTest.kt [hunk 2]
```diff
@@ -38,17 +38,17 @@ internal class IntegrationTest : Spek({
...
-                cfg
+                lug
             )
         }
     }
 
     describe("the integration of different components of the package for classes containing if statements") {
-        it("converts a class containing an if with a library usage in the false-branch to a filtered cfg") {
-            val cfg = LibraryUsageGraphGenerator.generate(
+        it("converts a class containing an if with a library usage in the false-branch to a filtered lug") {
```
Comment (gandreadis) [inline modules/pipeline/jimple-library-usage-graph-generator/src/test/kotlin/org/cafejojo/schaapi/pipeline/usagegraphgenerator/jimple/IntegrationTest.kt]: Perhaps `lug` -> `LUG`? I know the old version was all-lowercase, but this isn't readable for anyone not familiar with the codebase

Comment (casperboone) [reply modules/pipeline/jimple-library-usage-graph-generator/src/test/kotlin/org/cafejojo/schaapi/pipeline/usagegraphgenerator/jimple/IntegrationTest.kt]: I get what you mean, but I don't really like to capitalize local variables tbh. However, I've just changed it to libraryUsageGraph.

File: modules/pipeline/jimple-library-usage-graph-generator/src/main/kotlin/org/cafejojo/schaapi/pipeline/usagegraphgenerator/jimple/ControlFlowGraphGenerator.kt (hunk 1 above)
Comment (casperboone) [reply modules/pipeline/jimple-library-usage-graph-generator/src/main/kotlin/org/cafejojo/schaapi/pipeline/usagegraphgenerator/jimple/ControlFlowGraphGenerator.kt]: Yep, not really a part of this PR, but I've changed that as well :)

**Pull request merged**